includes/Admin/Admin_Page.php

```diff
@@ -150,16 +150,23 @@ private function get_available_plugins() {
 	 * Renders the "Plugin Check" page.
 	 *
 	 * @since n.e.x.t
+	 *
+	 * @global array  $available_plugins        The list of available plugins.
+	 * @global string $selected_plugin_basename The selected plugin basename.
+	 * @global array  $categories               An array of categories.
+	 * @global mixed  $user_settings            The user interface setting value.
 	 */
 	public function render_page() {
-		global $available_plugins, $selected_plugin_basename, $categories;
+		global $available_plugins, $selected_plugin_basename, $categories, $user_settings;
```

Comment:
Not strictly related to this PR, but do these actually need to be globals? If they are only used in the `admin-page.php` template that is `require_once`d right below in this same method, I think we should be able to use simply the local variables?

If that's the case, we should remove the `global` from this line to not pollute global scope unnecessarily.

Reply:
Thank you for bringing this up. PHPMD flagged the error, so in the commit https://github.com/WordPress/plugin-check/pull/304/commits/8a7bea21d60e18d0e5c9bbbb9149d1828ca9ba17, I removed the global variable and added `SuppressWarnings` annotations in the function.